bin/mail/2016-11-15-hoc-global-minecraft/send-real-emails

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+# 750550 teachers
+../send-to-mailing-list --limit 751000 2016-11-15-hoc-global-minecraft ./teachers.csv
```

Comment:
And sure enough, the new total is 752294. Since I haven't merged yet I'll update this in the PR as well.